package.json

```diff
@@ -2,31 +2,31 @@
   "name": "promjs",
   "version": "0.3.3",
   "description": "",
-  "main": "index.js",
+  "main": "dist/src/index.js",
+  "typings": "dist/src/index.d.ts",
```

Comment:
These should be just `dist/xxx.{js,d.ts}`, `dist/src` is confusing and unnecessary.